### ISSUE TITLE
need guidance on interior scene using custom data with MonoSDF

### ISSUE THREAD

**Issue by DavidNP96** [autonomousvision/sdfstudio#85]
Hi all,
I am trying to create an indoor scene, however, viewer results remain blurry (as per the image below). When I am importing the object into unreal I get a blob not even closely resemble the interior scene. I feel like I might be doing multiple things wrong in creating this result maybe someone here can help?

steps to recreate:
1. see data here https://drive.google.com/drive/folders/1EUspzF3vwK18N6kOoe4Cj6x3dG6JxZbB?usp=sharing
run following commands:
2. ns-process-data images --data data/sdfstudio-demo-data/${image_map_naam}/ --output-dir data/sdfstudio-demo- 
 data/{image_map_naam}/
3. python scripts/datasets/process_nerfstudio_to_sdfstudio.py   --data data/sdfstudio-demo-data/${image_folder_name}/   --output-dir data/sdfstudio-demo-data/${image_folder_name}/processed   --scene-type indoor   --data-type colmap   --mono-prior   --crop-mult 1   --omnidata-path  ./omnidata/omnidata_tools/torch  --pretrained-models  ./pretrained_models
4. ns-train monosdf --pipeline.model.sdf-field.inside-outside True --pipeline.model.mono-depth-loss-mult 0.1 --pipeline.model.mono-normal-loss-mult 0.05 --vis viewer --experiment-name neus-facto-replica1 sdfstudio-data --data data/sdfstudio-demo-data/conf/processed --include_mono_prior True

results in viewer:
![image](https://user-images.githubusercontent.com/44022245/232754747-618634de-1ed1-4cbd-aab6-9d34e77218ba.png)

results in unreal:
![image](https://user-images.githubusercontent.com/44022245/232760136-d1d00f3d-3739-450f-9f36-a73409af1cee.png)

**Comment by niujinshuchong** [comment]
@DavidNP96 Your dataset looks difficult. Could you get some reasonable results with `nerfacto`?

**Comment by DavidNP96** [comment]
thank you for your response @niujinshuchong the result of nerfacto is quite alright. However, there are quite a number of artifacts.

https://user-images.githubusercontent.com/44022245/233923055-81890a39-c69e-42f2-a370-de5228a14b7c.mp4

**Comment by niujinshuchong** [comment]
@DavidNP96 Nerfacto's result looks good. Maybe you could try to train a volsdf/neus model without monocular prior first to check how the results look like before adding monocular prior.

**Comment by DavidNP96** [comment]
@niujinshuchong 

This is the nerf result when extracting its mesh (using poisson meshing):

https://user-images.githubusercontent.com/44022245/234225720-f55ce348-e5dc-47c9-aa71-b7a7b316da1c.mp4

This is volSDF result without monocular priors :

https://user-images.githubusercontent.com/44022245/234226034-cc0afe80-51e7-4c41-adf3-7a091e45fffa.mp4

**Comment by niujinshuchong** [comment]
@DavidNP96 Looks like it's a difficult dataset. Maybe you could try other method in the repo, e.g. bakedsdf and bakedsdf-mlp. But I am not sure whether it will produce reasonable result or not.
